evap/staff/tests/test_views.py

```diff
@@ -1733,6 +1752,17 @@ def test_copy_participants(self):
 
         self.assertEqual(self.evaluation.participants.count(), original_participant_count + self.evaluation2.participants.count())
 
+    def test_copy_participants(self):
+        page = self.app.get(self.url, user=self.manager)
+
+        original_participant_count = self.evaluation.participants.count()
```

Comment:
you are not using this value again. maybe make sure that is is 0?
that would help to understand the `assertEqual` below. right now it is weird because one could think that copying should not make the number equal in contrast to the replacement operation. it is only true here, because `self.evaluation` didn't have any participants before.